.github/workflows/wolfi-presubmit.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
           - fping
           - subversion
           - sudo
+          - py3-supported-python
```

Comment:
Was this intended to be part of the change?

Reply:
Yeah, this was the only "buildless" package in Wolfi, and I added it here to make sure it works going forward